packages/core/src/components/legend-item/readme.md

```diff
@@ -0,0 +1,50 @@
+# dso-legend-item
```

Comment:
De heading1 in de readme files is altijd implementatie specifiek:

* `<dso-legend-item>`
* `.alert`
* `<DsoModal>`

In de plop generator zie ik dat we dat wel genereren, enig idee hoe dit is verdwenen?

Reply:
geen idee, wel hersteld